.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/compilerla/conventional-pre-commit
```

Comment:
question: should this be `distribution/distribution`? I'm reading this but I might be missing something https://github.com/compilerla/conventional-pre-commit#usage

Reply:
No, that's the repo where `pre-commit` is getting the source code for `conventional-pre-commit`. You can also use `pre-commit` to manage all sorts of other git hooks and their versions.

Reply:
ah, so that's basically saying: code in repo X will make the checks. Well that's not clear from their README 🫠

Reply:
I'm not really a huge fan of this solution for managing git hooks. But all the other ones I've found that look better are really focused on Node based projects and I'm not sure they'd integrate nicely. I did find a way it could be done using Go, but either we'd add additional dependencies to the project or I'd need to create a new tool that we can then use (which I might down the line).